assets/company-admin/components/CompanyDetails.jsx

```diff
@@ -30,14 +29,21 @@ function CompanyDetailsComponent({company, showSeatRequestModal, setSection, com
                             <tr colSpan={2 + numSections} className="subheading">
                                 <td>{section.name}</td>
                             </tr>
-                            {Object.values(seats[company._id]).filter((seat) => seat.section === section._id).map((seat) => (
-                                <CompanyDetailsProductRow
-                                    key={seat._id}
-                                    seat={seat}
-                                    showSeatRequestModal={showSeatRequestModal}
-                                    onNameClicked={() => setSection('users', seat._id)}
-                                />
-                            ))}
+
+                            {
+                                (query === null ?
+                                    Object.values(products[company._id]).filter((product) => product.section === section._id) : 
+                                    Object.values(products[company._id]).filter((product) => product.section === section._id && product.name.toString().includes(query)) 
```

Comment:
It would be good to convert the query and product.name to lowercase when comparing these two